osgpkitools/osg-cert-revoke

```diff
@@ -3,7 +3,7 @@
 import sys
 import urllib
 import httplib
-import M2Crypto
+from M2Crypto import httpslib, SSL
```

Comment:
Good catch, I must've missed it in my big commit shuffle. New commit addresses it.